cachito/web/models.py

```diff
@@ -301,10 +301,15 @@ class RequestDependency(db.Model):
         db.Integer, db.ForeignKey("request.id"), autoincrement=False, index=True, primary_key=True
     )
     dependency_id = db.Column(
-        db.Integer, db.ForeignKey("package.id"), autoincrement=False, index=True, primary_key=True
+        db.Integer, db.ForeignKey("package.id"), autoincrement=False, index=True, primary_key=True,
```

Comment:
Could you please remove the comma you added here since it's not relevant to the change?